src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java

```diff
@@ -924,3 +924,32 @@ public void onPlayerLogin(PlayerLoginEvent event)
         TFM_ServerInterface.handlePlayerLogin(event);
     }
 }
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onEntityDamageByEntity(EntityDamageByEntityEvent event)
+    {
+        if (event.getEntity() instanceof Player)
+        {
+            if (event.getDamager() instanceof Player)
+            {
+                Player player = (Player) event.getDamager();
+                if (player.getGameMode() == GameMode.CREATIVE)
+                {
+                    TFM_Util.playerMsg(player, "NO GM / GOD PVP!", ChatColor.DARK_RED);
```

Comment:
@Ptatoz @LegendIsAwesomes How about: `ChatColor.GRAY + "Creative PVP is prohibited."`